src/components/structures/MatrixChat.tsx

```diff
@@ -1388,6 +1389,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
 
         StorageManager.tryPersistStorage();
 
+        navigator.registerProtocolHandler("matrix", "/#/%s");
```

Comment:
![image](https://user-images.githubusercontent.com/2403652/152140948-c7bcaa5e-9935-4156-8b59-3c1e1a742e53.png)


This may want to happen on an in-app user interaction (Toast?) so the user understands the impact, will need @matrix-org/product input

Reply:
We'd like to be consistent with other areas of the app and have a toast first. Redirecting to design for input